### PR TITLE
Added in `www` to manifest.

### PR DESCRIPTION
**Pull request by adamharrison**
Adds in a library for HTTP(s) requests.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1370,6 +1370,14 @@
       "path": "plugins/wordcount.lua",
       "id": "wordcount",
       "mod_version": "3"
+    },
+    {
+      "id": "www",
+      "description": "A simple library that provides a web client to fetch, and submit data via HTTP requests.",
+      "version": "0.1",
+      "type": "library",
+      "remote": "https://github.com/adamharrison/lite-xl-www.git:aecc866c33b0a954fbd41a5de2842bb97f4ac45e",
```
Comment (Guldoman) [inline manifest.json:1379]: Could the `.git` part of the URL cause issues?

Comment (adamharrison) [reply manifest.json:1379]: Don't see how; that is part of the official URL as provided by github.

Comment (Guldoman) [reply manifest.json:1379]: It's just that some plugins are using `.git`, others aren't. I guess it's fine.

Comment (adamharrison) [reply manifest.json:1379]: Odd. Github probably handles it the same either way, if it starts getting requests from a git client. But technically, the `.git` is official.

**Pull request merged**